docs/configuration.md

```diff
@@ -482,12 +482,13 @@ Example:
 ```
 
 #### Properties
-| Name | Type | Required | Default |
-| ---- | ---- | -------- | ------- |
-| subreddit | string | yes |  |
+| Name | Type | Required | Default       |
+| ---- | ---- | -------- |---------------|
+| subreddit | string | yes |               |
 | style | string | no | vertical-list |
-| limit | integer | no | 15 |
-| collapse-after | integer | no | 5 |
+| limit | integer | no | 15            |
+| collapse-after | integer | no | 5             |
+| proxy | string | no |               |
```

Comment:
Was the extra spacing here added by mistake?

Reply:
Hey, it was probably added by my IDE, I'll revert these lines.